src/dispatch/organization/service.py

```diff
@@ -0,0 +1,44 @@
+from typing import List, Optional
+
+from fastapi.encoders import jsonable_encoder
+
+from .models import Organization, OrganizationCreate, OrganizationUpdate
+
+
+def get(*, db_session, organization_id: int) -> Optional[Organization]:
+    return db_session.query(Organization).filter(Organization.id == organization_id).first()
+
+
+def get_all(*, db_session) -> List[Optional[Organization]]:
+    return db_session.query(Organization)
+
+
+def create(*, db_session, organization_in: OrganizationCreate) -> Organization:
+    team = Organization(
+        **organization_in.dict(),
+    )
+    db_session.add(team)
+    db_session.commit()
+    return team
+
+
+def update(
+    *, db_session, organization: Organization, organization_in: OrganizationUpdate
+) -> Organization:
+    organization_data = jsonable_encoder(organization)
+
+    update_data = organization_in.dict(skip_defaults=True)
+
+    for field in organization_data:
+        if field in update_data:
+            setattr(organization, field, update_data[field])
+
+    db_session.add(organization)
+    db_session.commit()
+    return organization
+
+
+def delete(*, db_session, organization_id: int):
```

Comment:
Are we going to keep or delete users linked to the organization? Keep them and allow the admin to assign them to a new organization?

Reply:
We could go either way. I don't think it matters too much. But I would guess if we were to delete an organization it would delete all of the user information as well.

Reply:
That would be my preference too, but I also don't have a strong opinion.